src/setupTest.ts

```diff
@@ -20,6 +21,6 @@ class ResizeObserver {
 
 window.ResizeObserver = ResizeObserver;
 
-window.crypto = {
-  getRandomValues: arr => crypto.randomBytes(arr.length),
+(window as any).crypto = {
+  getRandomValues: (arr: any) => crypto.randomBytes(arr.length),
```

Comment:
@adamstankiewicz I see you added this `getRandomValues` stub in https://github.com/openedx/paragon/commit/eac3925bcd7710e711f3988a504834f28a93060e , but I noticed just now that the test suite passes just fine if I completely remove this line. Is it still necessary/useful?